docs/status.md

```diff
@@ -92,7 +92,7 @@ one of the following tracking labels.
 | fft                      | yes           | revisit      | yes            | yes             | no          |
 | floor                    | yes           | yes          | yes            | yes             | yes         |
 | gather                   | yes           | yes          | yes            | no              | no          |
-| get_dimension_size       | no            | yes*         | yes*           | yes             | no          |
```

Comment:
Shouldn't the type inference be yes

https://github.com/openxla/stablehlo/blob/93e7cf81bf8258021ef5d96c31682ab893cb82da/stablehlo/dialect/StablehloOps.cpp#L1267

Reply:
updated.